src/animations/Slide/Slide.js

```diff
@@ -23,35 +27,39 @@ class Slide extends PureComponent {
      * direction of where children would slide to e.g. "top right"
      */
     direction: PropTypes.string,
-    animationName: PropTypes.string,
     style: PropTypes.object,
   };
 
+  static defaultProps = {
+    direction: 'top right',
```

Comment:
Using commented example values provided in description of `propType`s. If you have a preferred value I'd gladly set that.

Reply:
I don't have the preferred value, I was just wondering because it is possible we're changing the component's default behavior.

When you change
```
const { inputRange = [0, 1] } = this.props;
```
into default prop, the behavior stays the same.

But, if the `direction` wasn't `top right` before and now is, it changes the default behavior. 

I don't think it makes any difference here (because I expect whoever used the `Slide` component had to provide some value) but I just wanted to point that out.